processing/src/main/java/io/druid/segment/incremental/OnheapIncrementalIndex.java

```diff
@@ -139,6 +145,7 @@ protected DimDim makeDimDim(String dimension, Object lock)
   protected Integer addToFacts(
       AggregatorFactory[] metrics,
       boolean deserializeComplexMetrics,
+      boolean reportParseExceptions,
```

Comment:
i cant see any benefits of passing it multiple times, applies to both the flags,
I am fine cleaning them up in separate PR, can you either clean them up or file an issue to track this ?